index.ftd

```diff
@@ -1,51 +1,1617 @@
 -- import: fastn-community.github.io/set-2-ds/assets
+-- import: fastn/processors as pr
+-- import: fastn-community.github.io/set-2-ds/common
+-- import: fastn-community.github.io/dark-mode-switcher as dms
+-- import: fastn-community.github.io/set-2-ds/header as h
+-- import: fastn-community.github.io/set-2-ds/common/fonts as fonts
 
 
 
--- ds.page: Welcome to your [FASTN site](https://fastn.io/)
+
+
+
+
+
+
+
+
+-- page: Documentation
+sidebar: true
+show-footer: true
+login-button: true
```

Comment:
```suggestion
login-button: true
```
@MeenuKumari28 this is not part of Set-1-DS while exchange from Set-1 to Set-2 it will break.